e2e/stack/openfire/embedded-db/openfire.script

```diff
@@ -155,6 +157,8 @@ INSERT INTO OFSECURITYAUDITLOG VALUES(4,'admin',1523539578286,'created new user
 INSERT INTO OFSECURITYAUDITLOG VALUES(5,'admin',1523626518432,'set server property httpbind.client.cert.policy','openfire','httpbind.client.cert.policy = needed')
 INSERT INTO OFSECURITYAUDITLOG VALUES(6,'admin',1523626524612,'set server property sasl.mechs','openfire','sasl.mechs = EXTERNAL')
 INSERT INTO OFSECURITYAUDITLOG VALUES(7,'admin',1525714848566,'destroyed PubSub Node collection1','openfire','reason = null')
+INSERT INTO OFSECURITYAUDITLOG VALUES(8,'admin',1528754284922,'set server property xmpp.httpbind.client.requests.max','openfire','xmpp.httpbind.client.requests.max = 200')
+INSERT INTO OFSECURITYAUDITLOG VALUES(9,'admin',1528754294762,'set server property xmpp.httpbind.client.requests.polling','openfire','xmpp.httpbind.client.requests.polling = 200')
```

Comment:
What config did you change here? Can we document this?

Reply:
I added a section to Openfire section in the install docs.